### ISSUE TITLE
cannot do validation

### ISSUE THREAD

**Issue by yuqwang913** [swook/GazeML#49]
Hi. I am trying to load a validation set and my data setting is shown below:

        validation_data = UnityEyes(
            session,
            batch_size=batch_size,
            data_format='NCHW',
            
 unityeyes_path='/cluster/project/infk/courses/machine_perception_19/validation_data/imgs_small/imgs',
            testing = True,
            num_threads=2,
            generate_heatmaps=True,
            eye_image_shape=(36, 60),
            heatmaps_scale=1.0 / elg_first_layer_stride,
        )
        validation_data.set_augmentation_range('translation', 2.0, 10.0)
        validation_data.set_augmentation_range('rotation', 1.0, 10.0)
        validation_data.set_augmentation_range('intensity', 0.5, 20.0)
        validation_data.set_augmentation_range('blur', 0.1, 1.0)
        validation_data.set_augmentation_range('scale', 0.01, 0.1)
        validation_data.set_augmentation_range('rescale', 1.0, 0.5)
        validation_data.set_augmentation_range('num_line', 0.0, 2.0)
        validation_data.set_augmentation_range('heatmap_sigma', 7.5, 2.5)  

But the issue is that the model cannot do validation job during training and it cannot trigger final test either. I want to ask whether there are some other settings I ignored if I wanna do validation during training? Thank you!
